pygeodiff/geodifflib.py

```diff
@@ -435,6 +435,19 @@ def read_changeset(self, changeset):
             raise GeoDiffLibError("Unable to open reader for: " + changeset)
         return ChangesetReader(self, reader_ptr)
 
+    def get_wkb_from_geometry(self, geometry):
+        func = self.lib.GEODIFF_createWkbFromGpkgHeader
+        func.argtypes = [ctypes.c_char_p, ctypes.POINTER(ctypes.c_size_t)]
+        func.restype = ctypes.POINTER(ctypes.c_char)
+        free_func = self.lib.GEODIFF_free
+        free_func.argtypes = [ctypes.c_char_p]
+        free_func.restype = None
```

Comment:
I would registerr GEODIFF_free function in self. so we can use it on other places (in function register_functions)